README.md

```diff
@@ -146,6 +146,60 @@ optional arguments:
                         DynamoDB table to use for credential storage
 ```
 
+## IAM Policies
+
+### Secret Writer
+In order to put or write secrets to credstash, and you are not using KMS Key Grants or Key Policies, at least the following permissions are required:
```

Comment:
May want to wordsmith that a bit, maybe instead of 

"In order to put or write secrets to credstash, and you are not using KMS Key Grants or Key Policies, at least the following permissions are required"

something like

"You can put or write secrets to credstash by either using KMS Key Grants, KMS Key Policies, or IAM Policies. If you are using IAM Policies, the following IAM permissions are the minimum required to be able to put or write secrets"